docs/monitoring/debug-a-pod.md

```diff
@@ -0,0 +1,362 @@
+---
+title: How to debug a pod
+description: "Your pod doesn't run as expected. Are there any log files? Where? How could I debug a pod?"
```

Comment:
Why do you think is guide needs to be g/g?

Reply:
because I found there a similar (to me ) doc files like `shell-to-node` and `tail-log-file`.
What do you suggest about this document?

Reply:
I don't see this as valid argument. There is already similar guide in k8s https://kubernetes.io/docs/tasks/debug-application-cluster/debug-running-pod/, you can also try to move this one to https://github.com/kubernetes/website/.

Reply:
OK, I have updated the PR without these files